### PR TITLE
Add Fox Pet

### PR DESCRIPTION
**Pull request by NicholasJohansan**
Adds fox as mentioned in #324 and a snow (white) variant.  
Adds attribution the original author of the assets in the README  
orange_lie_8fps
white_run_8fps

### CODE REVIEW

**Comment by tonybaloney** [comment]
This is brilliant, thank you! the animations are gorgeous.

Please could you set the colours as `red` and `white` instead of `orange` and `white`. We already have red as a colour preset.

**Comment by NicholasJohansan** [comment]
Sure, I'll do that, and I also realised I didn't add the fox in alphabetical order with the rest of the pets so I'll fix that as well

**Comment by tonybaloney** [comment]
I also realised we'll be adding both foxes and chickens in the same release. This is going to be chaos :rofl: . @gulyapulya @MhouneyLH

**Comment by tonybaloney** [comment]
screenshot 2022-12-13 at 12 51 47
because the fox has quite a lot of padding in the GIFs, it comes out as tiny compared with other pets. You'll need to crop the GIF frames whilst maintaining the same aspect ratio.

File: src/panel/pets.ts [hunk 1]
```diff
@@ -177,6 +178,8 @@ export function createPet(
             return new Chicken(...standardPetArguments, PetSpeed.normal);
         case PetType.dog:
             return new Dog(...standardPetArguments, PetSpeed.normal);
+        case PetType.fox:
+            return new Fox(...standardPetArguments, PetSpeed.veryFast);
```
Comment (tonybaloney) [inline src/panel/pets.ts:182]: Adjust this to fast, the animation doesn't keep up with the rate of movement.

**Comment by gulyapulya** [comment]
Btw, @NicholasJohansan, did you create the white one yourself? How did you change the color? Did you use filter? I am trying to create more colors for chicken, so looking for techniques.

**Comment by NicholasJohansan** [comment]
> Btw, @NicholasJohansan, did you create the white one yourself? How did you change the color? Did you use filter? I am trying to create more colors for chicken, so looking for techniques.

Yea, I created the white one myself. In Aseprite, it's possible to change the color pallete, so I just edited the shades of orange colors to be shades of white.

**Comment by NicholasJohansan** [comment]
done, hopefully the cropping is ok

**Comment by tonybaloney** [comment]
much better (although the chicken now seems massive!)

**Review by tonybaloney — APPROVED** [review]

**Comment by MhouneyLH** [comment]
Then I worry about the chickens.😂

**Comment by gulyapulya** [comment]
@tonybaloney, can we enlarge the fox to be at least the size of a dog or a cat? Or should we make chicken smaller?

**Comment by MhouneyLH** [comment]
> @tonybaloney, the fox still seems small to me, shouldn't we enlarge the fox to be at least the size of a dog or a cat? 😅 Or if you decide, then we can make chicken smaller.

I agree with this. It seems like the chicken could eat the fox in this version. 😂

**Comment by NicholasJohansan** [comment]
I think the chicken is the one that's too big, from the gif, the fox looks like the same size as the cat
image

**Pull request merged**